sdl_ios/SmartDeviceLink/SDLProtocol.m

```diff
@@ -14,21 +15,22 @@
 #import "SDLRPCPayload.h"
 #import "SDLDebugTool.h"
 #import "SDLPrioritizedObjectCollection.h"
+#import "SDLDataStreamHandlingDelegate.h"
 
 
-const NSUInteger MAX_TRANSMISSION_SIZE = 512;
+const NSUInteger MAX_TRANSMISSION_SIZE = 1024;
```

Comment:
Why was this increased, why wasn't it increased before, is there a limit we should be aware of? Where does this number come from?